repository_files.go

```diff
@@ -112,7 +112,7 @@ func (s *RepositoryFilesService) GetFileMetaData(pid interface{}, fileName strin
 	u := fmt.Sprintf(
 		"projects/%s/repository/files/%s",
 		pathEscape(project),
-		url.PathEscape(fileName),
+		pathEscape(url.PathEscape(fileName)),
```

Comment:
No need to call `url.PathEscape` twice:

```suggestion
		pathEscape(fileName),
```